src/TesterInternal/StorageTests/PersistenceGrainTests_AzureStore.cs

```diff
@@ -376,61 +343,8 @@ public void Persistence_Perf_Write_Reread()
                 grainAzureTable => grainAzureTable.DoRead());
```

Comment:
is this `grainAzureTable` or `grainAzureStorage`? Not exactly sure what the test does though, so take this comment with a grain of salt